src/main/groovy/com/bmuschko/gradle/docker/tasks/image/Dockerfile.groovy

```diff
@@ -31,23 +31,37 @@ class Dockerfile extends DefaultTask {
 
     @TaskAction
     void create() {
+        Instruction fromInstruction = findFromInstruction()
         verifyValidInstructions()
 
         getDestFile().withWriter { out ->
+            out.println fromInstruction.build()
             getInstructions().each { instruction ->
-                out.println instruction.build()
+                if(instruction.keyword != 'FROM'){ //FROM instruction already executed
+                    out.println instruction.build()
+                }
+            }
+        }
+    }
+
+    private Instruction findFromInstruction(){
+        Instruction fromInstruction
+        for(Instruction instruction: instructions){
+            if(instruction.keyword == 'FROM'){
+                fromInstruction = instruction
```

Comment:
My idea was using the last from instruction, making it possible to override. I added also some extra docs to the from methods for this.